pkg/apis/scope/v0alpha1/zz_generated.openapi_violation_exceptions.list

```diff
@@ -1,4 +1,4 @@
-API rule violation: list_type_missing,github.com/grafana/grafana/pkg/apis/scope/v0alpha1,ScopeDashboardSpec,DashboardUIDs
+API rule violation: list_type_missing,github.com/grafana/grafana/pkg/apis/scope/v0alpha1,ScopeDashboardBindingSpec,DashboardUIDs
 API rule violation: list_type_missing,github.com/grafana/grafana/pkg/apis/scope/v0alpha1,ScopeSpec,Filters
```

Comment:
@bergquist -- the "list_type_missing" warning means k8s wants to know what type of list the filters are.  In this case I think we want:

```
	// +listType=set
	Filters []ScopeFilter `json:"filters"`
```

This is used when running server side apply.  See: https://kubernetes.io/docs/reference/using-api/server-side-apply/#merge-strategy

Reply:
Ah! yea I _think_ atomic makes sense. I dont expect people to only add filters but if they do that seems to the least confusing option.